Header header
detection_osnet/Accuracy accuracy
float64 frame_loss